framework/db/Query.php

```diff
@@ -844,6 +849,119 @@ public function addParams($params)
     }
 
     /**
+     * @var array aliases defined for tables used in this query. `tablename => alias`.
```

Comment:
>  Was not original issue about using same table twice inside single query?

that was the issue with joinWith() where you can specify the alias with the changes in the other PR.
If a table is used twice you have to use the alias yourself to refer to the correct table. This method can not help you with that.